sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/IRoutingMapProvider.java

```diff
@@ -22,9 +22,9 @@
         /// <param name="range">This method will return all ranges which overlap this range.</param>
         /// <param name="forceRefresh">Whether forcefully refreshing the routing map is necessary</param>
         /// <returns>List of effective partition key ranges for a collection or null if collection doesn't exist.</returns>
-        Mono<List<PartitionKeyRange>> tryGetOverlappingRangesAsync(String collectionResourceId, Range<String> range,
-                                                                   boolean forceRefresh /* = false */, Map<String, Object> properties);
+        Mono<Utils.ValueHolder<List<PartitionKeyRange>>> tryGetOverlappingRangesAsync(String collectionResourceId, Range<String> range,
```

Comment:
This changes already are already merged:
https://github.com/Azure/azure-sdk-for-java/pull/6312

Are there more changes? or the branch is stale? could you merge the source branch back to your branch